SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc

```diff
@@ -0,0 +1,326 @@
+===== PID - Patient Identification Segment
+The HL7 Patient Identification (PID) segment requires a mapping from the SDC patient context information to the PID segment fields.
+
+.Prerequisite of Valid Patient Context
+[NOTE#ref_valid_pt_ctxt_note]
+====
+The SDC patient context information shall only be mapped to the corresponding fields in the HL7 PID segment when the requirements (e. g. inferred patient ensemble context, associated patient context, etc.) as defined in the <<ieee_11073_10700_2022>> are fulfilled.
+
+If the SDC patient context information shall not be used for the mapping, please refer to the <<ihe_pcd_tf-2_2019>> on how to populate the fields of the PID segment in this case.
+====
+
+====== PID-3 Patient Identifier List
+The PID-3 field is a list of patient identifiers (e. g. medical record number, socal security number, visit number, account number, etc.). The table <<ref_tbl_pid3_mapping>> defines the mapping of the SDC patient identifier to the data fields of the HL7 data type *CX* used in the PID-3 field.
+
+The SDC patient identifier element *pm:PatientContextState/pm:Identification* may contain more than one patient identifier. Each SDC patient identifier shall be mapped according to the <<ref_tbl_pid3_mapping>> table and added to the PID-3 patient identifier list.

```

Comment:
```suggestion
The BICEPS patient identification element (pm:PatientContextState/pm:Identification) may contain more than one patient identifier. Each SDC patient identifier shall be mapped according to the <<ref_tbl_pid3_mapping>> table and added to the PID-3 patient identifier list.
```

Reply:
This should be splitted into two requirements, one may, one shall.

Reply:
done. I kept the bold presentation since I used that everywhere - just to be consistent. Rephrased the sentence to a conditional shall in order to avoid a may requirement.